README.md

```diff
@@ -72,8 +67,95 @@ mjAPI.typeset({
+  extensions: '', // a convenience option to add MathJax extensions
+  fontURL: 'https://cdn.mathjax.org/mathjax/latest/fonts/HTML-CSS', // for webfont urls in the CSS for HTML output
+  MathJax: { } // standard MathJax configuration options, see https://docs.mathjax.org for more detail.
+}
+```
+
+**Note.** Changes to these options require a restart of the API using the `start()` method (see below).
+
+### `start()`
 
-The examples in `./bin/` provide a starting point for more advanced integrations.
+The `start` method start (and restarts) mathjax-node. This allows reconfiguration.
 
-Be sure to also check out other [projects on NPM that depend on mathjax-node ](https://www.npmjs.com/browse/depended/mathjax-node).
+**Note.** This is done automatically when `typeset` is first called (see below).
+
+### `typset(options, callback)`
+
+The `typeset` method is the main method of mathjax-node. It expects a configuration object and a callback.
+
+Once started, `typeset` can be called repeatedly and will potentially store information across calls (cf. `state` below).
+
+The following are the default options.
+
+```javascript
+{
+  ex: 6,                          // ex-size in pixels
+  width: 100,                     // width of container (in ex) for linebreaking and tags
+  useFontCache: true,             // use <defs> and <use> in svg output?
+  useGlobalCache: false,          // use common <defs> for all equations?
+  linebreaks: false,              // automatic linebreaking
+  equationNumbers: "none",        // automatic equation numbering ("none", "AMS" or "all")
+
+  math: "",                       // the math string to typeset
+  format: "TeX",                  // the input format (TeX, inline-TeX, AsciiMath, or MathML)
+  xmlns: "mml",                   // the namespace to use for MathML
+
+  html: false,                    // generate HTML output
+  htmlNode: false,                // generate HTML output as jsdom node
+  css: false,                     // generate CSS for HTML output
+  mml: false,                     // generate MathML output
+  mmlNode: false,                 // generate MathML output as jsdom node
+  svg: false,                     // generate SVG output
+  svgNode: false,                 // generate SVG output as jsdom node
+
+  speakText: true,                // add textual alternative (for TeX/asciimath the input string, for MathML a dummy string)
+
+  state: {},                    // an object to store information from multiple calls (e.g., <defs> if useGlobalCache, counter for equation numbering if equationNumbers ar )
+  timeout: 10 * 1000,             // 10 second timeout before restarting MathJax
+}
+```
+
+### `callback(result, input)`
+
+mathjax-node returns two objects to the `callback`, a return object and the original configuration object.
+
+The result (at most) the following structure:
```

Comment:
Did the original `data` element ever get added to the returned result?  If so, it should be documented below.